src/marten/cli/manage/command/collect_assets.cr

```diff
@@ -1,3 +1,5 @@
+require "digest/md5"
```

Comment:
Marten already requires `digest/md5`, so we don't need to add this requirement here! 😃